core/storage/ipld/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+#
+#Copyright Soramitsu Co., Ltd.All Rights Reserved.
+#SPDX - License - Identifier : Apache - 2.0
+#
+
+find_package(Protobuf REQUIRED)
+set(PB_SCHEME "ipld_node.proto")
+set(PB_BUILD_DIR ${CMAKE_CURRENT_BINARY_DIR})
+
+execute_process(COMMAND ${PROTOBUF_PROTOC_EXECUTABLE} --cpp_out=${PB_BUILD_DIR} ${PB_SCHEME}
```

Comment:
There is another protobuf-dependent module, pls add task for refactoring protobuf-generated modules to use kagome-like scheme significantly simplifying them.

Reply:
.+ please check https://github.com/filecoin-project/cpp-filecoin/pull/114

Reply:
FIL-150